src/app/researcher/study/request/[memberIdentifier]/study-proposal.tsx

```diff
@@ -27,100 +39,113 @@ export const StudyProposalForm: FC<{
-                    <TextInput aria-label="Principal Investigator" {...studyProposalForm.getInputProps('piName')} />
-                </Group>
-
-                <Group>
-                    <Text>Study Description</Text>
-                    {fileUpload}
-                    <Stack gap={0}>
-                        <FileInput
-                            name="descriptionDocument"
-                            component={Anchor}
-                            aria-label="Upload Study Description Document"
-                            placeholder="Upload Document"
-                            clearable
-                            accept=".doc,.docx,.txt,.pdf"
-                            {...studyProposalForm.getInputProps('descriptionDocument')}
-                        />
-                        <Text size="xs" c="dimmed">
-                            Accepted formats: doc, docx, pdf
-                        </Text>
-                    </Stack>
-                </Group>
-
-                <Group gap="xs">
-                    <Text>IRB Document</Text>
-                    {irbFileUpload}
-                    <Stack gap={0}>
-                        <FileInput
-                            name="irbDocument"
-                            color="purple"
-                            component={Anchor}
-                            aria-label="Upload IRB Document"
-                            placeholder="Upload Document"
-                            clearable
-                            accept=".doc,.docx,.txt,.pdf"
-                            key={studyProposalForm.key('irbDocument')}
-                            {...studyProposalForm.getInputProps('irbDocument')}
-                        />
-                        <Text size="xs" c="dimmed">
-                            Accepted formats: doc, docx, pdf
-                        </Text>
-                    </Stack>
-                </Group>
-
-                <Group gap="xs">
-                    <Text>Agreement Document</Text>
-                    {agreementFileUpload}
-                    <Stack gap={0}>
-                        <FileInput
-                            name="agreementDocument"
-                            aria-label="Upload Agreement Document"
-                            component={Anchor}
-                            placeholder="Upload Document"
-                            clearable
-                            accept=".doc,.docx,.txt,.pdf"
-                            key={studyProposalForm.key('agreementDocument')}
-                            {...studyProposalForm.getInputProps('agreementDocument')}
-                        />
-                        <Text size="xs" c="dimmed">
-                            Accepted formats: doc, docx, pdf
-                        </Text>
-                    </Stack>
-                </Group>
-            </Stack>
+            <Flex>
+                <Table variant="vertical" withRowBorders={false}>
```

Comment:
I think using a <Table> for the form layout is a little overkill and can probably be done simpler 🤔 maybe something similar to how the `StudyDetails` component lays everything out 🤔